Lib/test/test_socket.py

```diff
@@ -1538,6 +1538,31 @@ def testSetSockOpt(self):
         reuse = sock.getsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR)
         self.assertFalse(reuse == 0, "failed to set reuse mode")
 
+    @unittest.skipIf(_testcapi is None, "requires _testcapi")
+    def test_setsockopt_errors(self):
+        # See issue #107546.
+        from _testcapi import INT_MAX, INT_MIN
```

Comment:
We could just use some large values like `2**1000` and `-2**1000`.

Reply:
I'm scared by big numbers, `2**100` and `-2**100` should be enough to trigger OverflowError on all platforms :-)

Reply:
But not on platforms with 128-bit `int`. I often use `2**1000` as more future proof option.